src/clarity-angular/wizard/_wizard.clarity.scss

```diff
@@ -354,7 +354,7 @@ $clr-wizard-box-shadow: 0 $clr-rem-1px (2 * $clr-rem-1px) (2 * $clr-rem-1px) rgb
             }
         }
 
-        .spinner:not(.spinner-inline) {
+        .spinner:not(.spinner-inline):not(.clr-treenode-spinner) {
             left: calc(50% + 4.791667rem);
```

Comment:
@adityarb88 @mathisscott this `4.791667rem` is so specific. Should we assign it to a variable?

Reply:
@Shijir: We could but its not related to this PR. We can revisit this later on.